infra/compile.py

```diff
@@ -765,15 +766,14 @@ def compile_module(tree, name, use_js_modules):
 
         imports_str = "import {{ {} }} from \"./{}\";"
 
-        rt_imports_arr = [ 'breakpoint', 'pysplit', 'truthy' ]
-        rt_imports = imports_str.format(",".join(rt_imports_arr), "rt-module.js")
+        rt_imports_arr = [ 'breakpoint', 'comparator', 'filesystem', 'pysplit',
+        'socket', 'ssl', 'tkinter', 'truthy' ]
```

Comment:
I think here we can do `[ 'breakpoint', 'comparator', 'filesystem', 'pysplit', 'truthy' ] + IMPORTS`. Honestly it does not matter much, but as we write more chapters it'll remove a hassle.